Makefile.am

```diff
@@ -162,7 +162,9 @@ systemdsystemunit_DATA += $(eos_updater_avahi_in:%.in=%)
 EXTRA_DIST += $(eos_updater_avahi_in)
 CLEANFILES += $(eos_updater_avahi_in:%.in=%)
 
-dist_man8_MANS += eos-updater-avahi/docs/eos-updater-avahi.8
+dist_man8_MANS += \
+	eos-updater-avahi/docs/eos-updater-avahi.8 \
+	eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8
```

Comment:
Nitpick: Please end the list with `$(NULL)` on a line of its own, so that future additions of man pages don’t cause diff noise by adding a backslash on the preceding line.